tron/api/resource.py

```diff
@@ -218,7 +229,7 @@ def __init__(self, job_collection):
     def getChild(self, name, request):
         if not name:
             return self
-        return resource_from_collection(self.job_collection, name, JobResource)
+        return resource_from_collection(self.job_collection, name.decode(), JobResource)
```

Comment:
We should probably be decoding the name in all getChild calls before we use it, not just here. I'm going by the "use bytes at the boundaries, unicode everywhere else" rule. 

And since b'job_name' != 'job_name', I anticipate issues elsewhere. For example, there are some comparisons to '_event, and I think the dictionary keys from state are going to be unicode, not bytes. The unit tests don't really cover this.

Reply:
yeah, you're right, i'll update other resources